src/js_command_match/transactions/status.rs

```diff
@@ -0,0 +1,70 @@
+use crate::js_command_match::constants::NETWORK_ID_ALIASES;
+
+#[derive(Debug, Clone, clap::Parser)]
+pub struct TxStatusArgs {
+    hash: String,
+    #[clap(allow_hyphen_values = true, num_args = 0..)]
+    _unknown_args: Vec<String>,
+    #[clap(long, aliases = NETWORK_ID_ALIASES, default_value=None)]
+    network_id: Option<String>,
```

Comment:
Is there a specific meaning for putting `_unknown_args` in the middle instead of the end of the fields? It not, let's move it down:

```suggestion
    #[clap(long, aliases = NETWORK_ID_ALIASES, default_value=None)]
    network_id: Option<String>,
    #[clap(allow_hyphen_values = true, num_args = 0..)]
    _unknown_args: Vec<String>,
```